### DISCUSSION TITLE
Plug internet-check presence

### DISCUSSION THREAD

**Discussion by pascalbourges** [netalertx/NetAlertX#880] [Q&A]
Hi, i Can see internet online but presence IS all empty, do i have to setup something ?
Other thing, in tools when i click internet speedtest, i have only a gray Line in result.

**Comment by jokob-sk** [comment]
Hi,

Could you please provide more details and screenshots? I'm not sure what you mean. 

Thanks,
j

**Comment by pascalbourges** [comment]
here device internet is always online
![image](https://github.com/user-attachments/assets/9b5c0121-be9a-4708-9324-0f7ee4bb9ba6)
But here, no indication of presence
![image](https://github.com/user-attachments/assets/3aec74b1-0107-4499-a30a-e924e9dad69c)

**Reply by jokob-sk** [reply]
Hi,

Thanks for that.

What version are you using? This should be fixed in the latest release.

https://github.com/jokob-sk/NetAlertX/releases/tag/v24.10.31 (under 💠Core app updates)

**Reply by pascalbourges** [reply]
I have v24.10.31

**Reply by jokob-sk** [reply]
Yeah, that's the latest stable. Hummm, not sure. I might have a look at it later, but it's very low priority as it's not impacting the core functionality. If  you code, feel free to have a look how this can be fixed. 

This thread might be helpful if you decide to have a look into it: https://github.com/jokob-sk/NetAlertX/issues/814
